core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/MetricsServiceImpl.java

```diff
@@ -940,11 +940,7 @@ public void setDefaultTTL(int defaultTTL) {
 
     @Override
     public Observable<Boolean> idExists(final MetricId<?> metricId) {
```

Comment:
I suspect it was used in one of the earlier versions of the JAX-RS implementations. Should we get rid of it? I see it as a useful method on this core service class.